webapp/sources/api-doc/paths/nodes/status.yml

```diff
@@ -1,18 +1,11 @@
 # SPDX-License-Identifier: CC-BY-SA-2.0
 # SPDX-FileCopyrightText: 2013-2020 Normation SAS
 get:
-  summary: Get status (pending, accepted, unknown) of a list of nodes
-  description: 
+  summary: Get nodes acceptation status
+  description: Get acceptation status (pending, accepted, deleted, unknown) of a list of nodes
   operationId: getNodesStatus
   parameters:
-    name: "ids"
```

Comment:
The problem was that parameters expects an array. I finally split it to allow easier reuse (and maybe better standardization of parameters).